pages/your_domain/structural_bioinformatics.md

```diff
@@ -1,36 +1,51 @@
 ---
 title: Structural Bioinformatics
 search_exclude: true
```

Comment:
```suggestion
```
If you are satisfied with this page you can delete this attribute :)

Reply:
Sure. I will commit the change.